articles/windows-mdm-setup.md

```diff
@@ -141,6 +143,37 @@ Testing automatic enrollment requires creating a test user in Microsoft Entra ID
 2. After it's been wiped, open your workstation and follow the setup steps. At screen in which you're asked to sign in, you should see the title "Welcome to [your organziation]!" next to the logo you uploaded in step 4.
 
 
+## Automatic Windows MDM Migration
+
+Fleet can automatically migrate your Windows hosts from a 3rd party MDM solution to Fleet MDM. Once configured, the migration happens quickly and doesn't require any end user interaction.
```

Comment:
```suggestion
Fleet can automatically migrate your Windows hosts from another MDM solution to Fleet without end user interaction.
```

Simplify language. "Fleet" instead of "Fleet MDM" because Fleet has MDM features. MDM features aren't a separate product.